### ISSUE TITLE
context.use does not return a context object

### ISSUE THREAD

**Issue by avgdev** [gabrieldemarmiesse/python-on-whales#476]
`docker.context.create` returns a `context` object which can be later used in a `DockerClient`
```
context = docker.context.create(
                    context, 
                    docker=DockerContextConfig(host=base_url)
                ) 

print(type(context))
<class 'python_on_whales.components.context.cli_wrapper.Context'>
```
However if the context already exists a `DockerException` is thrown:
```
context = docker.context.create(
                    context, 
                    docker=DockerContextConfig(host=base_url)
                ) 

python_on_whales.exceptions.DockerException: The docker command executed was `/.cache/python-on-whales/docker-cli/20.10.5/docker context create --docker host=ssh://root@192.168.64.11,skip-tls-verify=False 192.168.64.11`.
It returned with code 1
The content of stdout is ''
The content of stderr is 'context "192.168.64.11" already exists
```
Calling `docker.context.use("192.168.64.11")` only switches to the context but does not return a context object making it very difficult to create a `DockerClient` with a context.

**Comment by gabrieldemarmiesse** [comment]
I think a workaround would be to use `docker.context.inspect()` to get the Context object, which is itself a context manager. But indeed it would be nice if the context could be returned with `docker.context.use()`. I'm open to a pull request to return the object.

**Comment by gabrieldemarmiesse** [comment]
Fixed by #488
